app/javascript/hooks/request-query.js

```diff
@@ -3,7 +3,7 @@ import { UrlParams } from '../utils/url-params'
 import fetch from 'isomorphic-fetch'
 import { camelizeKeys } from 'humps'
 
-async function handleFetch(key, url, query) {
+function handleFetch(key, url, query) {
```

Comment:
We didn't use the `await` keyword, so the `async` modifier doesn't actually do anything.

Reply:
Ah yes!